README.md

```diff
@@ -34,4 +34,32 @@ that's built into gxi, so please don't use `cargo` to install gxi, as that won't
 from a different rev can lead to very weird bugs.
 
 
-After these steps you should be able to run gxi simply by invoking `gxi`
\ No newline at end of file
+After these steps you should be able to run gxi simply by invoking `gxi`
+
+### Installation on Arch/Manjaro
+
+There are two packages for gxi in Arch Linux's
+[AUR](https://aur.archlinux.org/). The first is the versioned package
+[gxi](https://aur.archlinux.org/packages/gxi/) and the second is the git
```

Comment:
Maybe "The first one follows gxi's release cycle"? Dunno, I'm fine with the first one too

Reply:
How about "The first is the regular release cycle package..."